doc/classes/ProjectSettings.xml

```diff
@@ -747,7 +747,7 @@
 			[b]Note:[/b] This setting is implemented only on macOS.
 		</member>
 		<member name="display/window/size/initial_position" type="Vector2i" setter="" getter="" default="Vector2i(0, 0)">
-			Main window initial position (in virtual desktop coordinates), this settings is used only if [member display/window/size/initial_position_type] is set to "Absolute" ([code]0[/code]).
+			Main window initial position (in virtual desktop coordinates), this setting is used only if [member display/window/size/initial_position_type] is set to "Absolute" ([code]0[/code]).
```

Comment:
```suggestion
			Main window initial position (in virtual desktop coordinates), only used if [member display/window/size/initial_position_type] is set to "Absolute" ([code]0[/code]).
```
Think more concise description is better here, and similar places

Reply:
This was just a typo fix. I'd rather not alter an unrelated doc further.